draft-ietf-quic-transport.md

```diff
@@ -827,6 +848,24 @@ Implementations MUST assume that an unsupported version uses an unknown packet
 format. All other fields MUST be ignored when processing a packet that contains
 an unsupported version.
 
+## Latency Spin Bit
+
+The latency spin bit enables latency monitoring from observation points on
+the network path. This bit is set as follow:
+
+* The connection responder sets the spin bit value to the value of the
+  spin bit in the last packet received from the connection initiator.
```

Comment:
"connection initiator" and "connection responder"  are just pretentious ways of saying client and server.  We happily use client-server protocols in non-client-server arrangements all the time.